compiler/src/iree/compiler/DispatchCreation/test/pipeline_tests_aggressive.mlir

```diff
@@ -0,0 +1,90 @@
+// RUN: iree-opt --split-input-file --pass-pipeline="builtin.module(util.func(iree-dispatch-creation-transpose-generic-ops, iree-dispatch-creation-elementwise-op-fusion,iree-dispatch-creation-form-dispatch-regions{aggressive-fusion=true}, iree-dispatch-creation-clone-producers-into-dispatch-regions, iree-dispatch-creation-elementwise-op-fusion{intra-dispatch=1 fuse-truncate-ops=1}, iree-dispatch-creation-collapse-dimensions, iree-dispatch-creation-convert-dispatch-regions-to-workgroups, canonicalize, cse))" --mlir-print-local-scope %s | FileCheck %s
```

Comment:
Instead of this, refer to `pipeline_test.mlir` in the other file and just use that. If you add a `clEnableFuseTruncateWithConsumers` you can just use that and make this easier.